### DISCUSSION TITLE
Feature request: init from remote source

### DISCUSSION THREAD

**Discussion by Arctomachine** [biomejs/biome#1339] [Ideas]
Problem: `biome init` creates basically empty config file. Developers who redefine defaults have to manually write/copy all their custom rules for each new project.

Solution: allow passing source (git repo or npm package) for config reference `biome init --source=@my-public-repo/biome-config`.

Things to consider:

- versioning - how to handle deprecated props when project installs newer version than remote file was made for and generally how to let user/maintainer know that it is outdated
- format - is it just ready made copy of config file or is it special key-value file that is parsed at init time to populate all valid props?
- splitting - ability to reference and merge multiple remote sources. For example, one with redefined formatter rules, another with redefined linter rules

**Comment by ematipico** [comment]
What would the configuration look like after the command?

**Reply by Arctomachine** [reply]
Normal biome.json file with all props from reference already in place, if you ask about that.
Unsure about schema version, as mentioned in original post, but I think it most certainly should be version of installed package.
